lib/root/widgets/cart_icon.dart

```diff
@@ -0,0 +1,35 @@
+import 'package:badges/badges.dart' as badges;
```

Comment:
hmmm we can remove the prefix no? it seems like it is just making the build method more verbose

Reply:
This prefix has to stay as Flutter Material has a `Badge()` widget now..